File: framework/languages/java/samples/java/TicTacToe.SessionGateway/src/main/java/systems/zlink/samples/tictactoe/sessiongateway/shared/configuration/SampleTopology.java
package systems.zlink.samples.tictactoe.sessiongateway.shared.configuration;

public final class SampleTopology {
    public static final String RegistryPubEndpoint = "tcp://127.0.0.1:19181";
    public static final String RegistryRouterEndpoint = "tcp://127.0.0.1:19182";
    public static final String PlayRouteEndpoint = "inproc://zlink-java-sample-session-play-route";
    public static final String ApiEndpoint = "tcp://127.0.0.1:47403";
    public static final String PlayEndpoint = "tcp://127.0.0.1:47404";
    public static final String SessionEndpoint = "tcp://127.0.0.1:47412";

    private SampleTopology() {
    }
}

File: framework/languages/java/samples/kotlin/TicTacToe.SessionGateway/src/main/kotlin/systems/zlink/samples/kotlin/tictactoe/sessiongateway/shared/configuration/SampleTopology.kt
package systems.zlink.samples.kotlin.tictactoe.sessiongateway.shared.configuration

object SampleTopology {
    const val RegistryPubEndpoint: String = "tcp://127.0.0.1:19191"
    const val RegistryRouterEndpoint: String = "tcp://127.0.0.1:19192"
    const val PlayRouteEndpoint: String = "inproc://zlink-kotlin-sample-session-play-route"
    const val ApiEndpoint: String = "tcp://127.0.0.1:47503"
    const val PlayEndpoint: String = "tcp://127.0.0.1:47504"
    const val SessionEndpoint: String = "tcp://127.0.0.1:47512"
}
